templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -930,6 +930,67 @@ export default {
         if (!validPreviousAnswers) {
           errMsgs.push("Please select a previous activity to display")
         }
+        const questionsWithDragDropEnabled = this.node.typeData.activity.questions.filter(
+          question => {
+            return question.answerTypes.dragDrop.enabled
+          }
+        )
+        const validFromBucketValues = questionsWithDragDropEnabled.every(
+          question => {
+            return question.answerTypes.dragDrop.fromBucketArray.every(
+              fromBucket => {
+                return fromBucket.value != ""
+              }
+            )
+          }
+        )
+        if (!validFromBucketValues) {
+          errMsgs.push("Please provide a text for all From Bucket label")
+        }
+        const validToBucketValues = questionsWithDragDropEnabled.every(question => {
+          return question.answerTypes.dragDrop.toBucketArray.every(toBucket => {
+            return toBucket.value != ""
+          })
+        })
+        if (!validToBucketValues) {
+          errMsgs.push("Please provide a text for all To Bucket label")
+        }
+        const validFromBucketItemValue = questionsWithDragDropEnabled.every(
+          question => {
+            return question.answerTypes.dragDrop.fromBucketArray.every(
+              fromBucket => {
+                return fromBucket.itemArray.every(fromBucketItem => {
+                  return fromBucketItem.text != ""
+                })
+              }
+            )
+          }
+        )
+        if (!validFromBucketItemValue) {
+          errMsgs.push("Please provide a text for all from bucket item(s) label")
+        }
+        const validFromBucketItemImage = questionsWithDragDropEnabled.every(
+          question => {
+            const dragDropUseImages = question.answerTypes.dragDrop.useImages
+            if (dragDropUseImages) {
```

Comment:
I generally don't think it's necessary to declare a constant if you are only going to use it once. Just put the original variable in your if statement.